options/conversion.yaml

```
@@ -0,0 +1,2 @@
+Options:
+  - IgnoreMaxLineLength: false
```

```suggestion
Validation:
  IgnoreMaxLineLength: false
```
Because the whole file are the options and the "-" marks an array in yaml but this isnt needed here

---

```
export class Options {
  Validation?: ValidationOptions = new ValidationOptions();
}

export class ValidationOptions {
  IgnoreMaxLineLength: boolean = false;
}
```

this would be the needed models